Rework Layered Attestation Environments intro (#256)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

## Layered Attestation Environments {#layered-attestation}

By definition, the Attester role generates Evidence.
An Attester may consist of one or
more nested environments (layers).

> The root layers of an Attester are called roots of trust.

Reviewer: ```suggestion
The root layer of an Attester encompasses the initial roots of trust included in an Attester
```

> In order to appraise Evidence produced by an Attester, the Verifier needs to trust the Attester's root of trust.

Reviewer: ```suggestion
In order to appraise Evidence generated by an Attester, the Verifier needs to trust the Attester's root of trust.
```

Trust in the Attester's root of trust can be established either directly (e.g., the Verifier puts the root of trust's public key into its trust anchor store) or transitively via an Endorser (e.g., the Verifier puts the Endorser's public key into its trust anchor store).
In layered attestation, a root of trust is the initial Attesting Environment.

> Claims can be collected from or about each layer, including roots of trust.

Reviewer: "Claims can be collected from or about each layer, including roots of trust."

This is technically correct. On the glossary level we have the responsibility to make absolutely sure that the reader is aware that a root of trust cannot ever generate Evidence from Claims about itself.

> Claims can be collected from or about each layer, including roots of trust.

Reviewer 2: ```suggestion
Claims can be collected from or about each layer.
```

## Layered Attestation Environments {#layered-attestation}

By definition, the Attester role generates Evidence.
An Attester may consist of one or
more nested environments (layers).
The root layers of an Attester are called roots of trust.
In order to appraise Evidence produced by an Attester, the Verifier needs to trust the Attester's root of trust.
Trust in the Attester's root of trust can be established either directly (e.g., the Verifier puts the root of trust's public key into its trust anchor store) or transitively via an Endorser (e.g., the Verifier puts the Endorser's public key into its trust anchor store).
In layered attestation, a root of trust is the initial Attesting Environment.
Claims can be collected from or about each layer, including roots of trust.
The corresponding claims can be structured in a nested fashion that reflects the nesting of the Attester's layers.

> 

Reviewer 3: Normally, claims are not self-asserted by a layer or root of trust.
